tools/dvrescue.xsd

```diff
@@ -32,7 +32,13 @@
     <xsd:sequence>
       <xsd:element name="frames" type="framesType" minOccurs="0" maxOccurs="unbounded"/>
     </xsd:sequence>
-    <xsd:attribute name="ref" type="xsd:string"/>
+    <xsd:attribute name="ref" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+        Reference to the original file path (relative).
```

Comment:
File path is either relative or absolute, actually it depends on user input, so no indication about that should be there.
If unique choice is needed, I think it is safer to use absolute path.